px-context-browser.html

```diff
@@ -1025,6 +1129,71 @@ <h1 class="epsilon caps weight--normal u-m0" on-click="toggleColumnBrowser">
         if (root && root.render) {
           root.render();
         }
+      },
+
+      /**
+       * Called when the checkboxes are clicked
+       * @private
+       *
+       */
+      _itemsSelected: function(evt) {
+        evt.stopPropagation();
+        if (evt.target.checked && this.limit !== 0 && this.limit === this.selectedItems.length) {
+          evt.preventDefault();
+          return false;
+        }
+
+        if (evt.target.checked) {
+          this.selectedItems.push(evt.model.item);
+        } else {
+          _.pull(this.selectedItems, evt.model.item);
+        }
+      },
+
+      /**
+       * Called when 'select all' checkbox is clicked
+       * @private
+       *
+       */
+      _selectAllItems: function(evt) {
+        const cls = evt.target.className;
+        const re = /level-[0-9]*/i;
+        const clsFound = cls.match(re);
+        let all = document.querySelector('span.'+clsFound);
+        const chkBoxes = all.querySelectorAll('.multi-select-chk:not([style*="display:none;"]):not([style*="display: none;"])');
+        let i = 0;
+        for (let i = 0; i < chkBoxes.length; i++) {
+          if (evt.target.checked && !chkBoxes[i].checked || !evt.target.checked && chkBoxes[i].checked) {
+            chkBoxes[i].click();
+          }
+        }
+      },
+
+      /**
+       *
+       * To show the message 'Selection limited to $limit'
+       * @returns {boolean}
+       * @private
+       */
+      _computeHidden: function(limit) {
```

Comment:
I'd suggest a less generic name for this, e.g. `_isSelectAllLimited`, that describes what you are checking.